libcontainer/configs/config_linux.go

```diff
@@ -13,10 +13,10 @@ var (
 // different when user namespaces are enabled.
 func (c Config) HostUID(containerId int) (int, error) {
 	if c.Namespaces.Contains(NEWUSER) {
-		if c.UidMappings == nil {
+		if c.UIDMappings == nil || len(c.UIDMappings) == 0 {
```

Comment:
Very tiny nit: this could be simplified to just;

```suggestion
		if len(c.UIDMappings) == 0 {
```

a `nil` slice has length `0`; https://go.dev/play/p/tSRvH2e-yH0

```go
package main

import "fmt"

func main() {
	var foo []int = nil
	if len(foo) == 0 {
		fmt.Println("YUP!")
	}
}
```

Reply:
No worries! There was nothing really _wrong_ with the old code, just not really needed, so where possible, I try to keep things simple 🤗